srv/modules/runners/proposal.py

```diff
@@ -308,6 +310,159 @@ def _record_filter(args, base_dir):
         yaml.dump(current_filter, filehandle, default_flow_style=False)
 
 
+def _find_minions_to_replace(profile_dir):
+    """
+    Search through a given profile directory for files that end with '-replace'.
+
+    :param profile_dir: Profile directory, e.g. "/srv/pillar/ceph/proposals/profile-default"
+    """
+    ToReplace = namedtuple('ToReplace', ['fullpath', 'filename'])
+    dir = '{}/stack/default/ceph/minions'.format(profile_dir)
```

Comment:
```dir``` is a python built-in function. maybe use ```dir_``` or ```_dir``` to avoid confusion?

Reply:
oh, right! Will change that.